requirements.txt

```diff
@@ -0,0 +1,2 @@
+Django>=2.0,<3.0
+django-widget-tweaks
```

Comment:
We could consider using `pip-sync` from piptools to have reproducible dependencies. (Though this may be too small of a project to fetch it in here)

Reply:
i think pinning exact versions would be quite catastrophic, because if the user is also using widget-twaks it have to be pinned to the exact same version everywhere

Reply:
Good point, exact pinning is probably not what we want for a library. Still, we may consider specifying an upper bound for all dependencies.